### PR TITLE
Story rework: VaViewer

### PR DESCRIPTION
**Pull request by xiongmao86**
Story rework of VaViewer
Part of #3676.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Improvement/refactoring (non-breaking change that doesn't add any feature but make things better)

### CODE REVIEW

File: packages/ui/src/components/va-viewer/VaViewer.stories.ts [hunk 1]
```diff
@@ -1,12 +1,96 @@
...
+
+Default.play = async ({ canvasElement, step }) => {
+  const viewer = canvasElement.querySelector('.va-viewer')
+  await step('Click on image to show a magnified view', async () => {
+    const beforeClick = canvasElement.querySelectorAll('.va-viewer-content')
+    expect(beforeClick.length).toBe(0)
+    userEvent.click(viewer)
+    await sleep()
+    const afterClick = canvasElement.querySelectorAll('.va-viewer-content')
+    expect(afterClick.length).toBe(1)
```
Comment (xiongmao86) [inline packages/ui/src/components/va-viewer/VaViewer.stories.ts:42]: I don't know why this won't work. I have tested with document.querySelector function and it doesn't work either.

**Comment by xiongmao86** [comment]
Ah, why is the chromatic test is not run? I have checked in local storybook and the one of the interaction test is not passed.

File: packages/ui/src/components/va-viewer/VaViewer.stories.ts (hunk 1 above)
Comment (asvae) [reply packages/ui/src/components/va-viewer/VaViewer.stories.ts]: Actucally document.querySelector worked just fine for me.

The issue was that viewer spawned outside of canvasElement.

image

**Review by asvae — APPROVED** [review]
Looks good!

I added a tweak for the failing test.

**Comment by xiongmao86** [comment]
Thank you very much for helping me with this pr, @asvae.

**Pull request merged**